openmc/plotter.py

```diff
@@ -0,0 +1,588 @@
+from numbers import Integral, Real
+
+import numpy as np
+
+import openmc.checkvalue as cv
+import openmc.data
+
+# Supported keywords for xs plotting
+PLOT_TYPES = ['total', 'scatter', 'elastic', 'inelastic', 'fission',
+              'absorption', 'capture', 'nu-fission', 'nu-scatter', 'unity',
+              'slowing-down power', 'damage']
+
+# Special MT values
+UNITY_MT = -1
+XI_MT = -2
+
+# MTs to combine to generate associated plot_types
+_INELASTIC = [mt for mt in openmc.data.SUM_RULES[3] if mt not in [5, 27]]
```

Comment:
I erroneously assumed it didnt have a neutron in the outgoing channel. Now I include it and test for the presence of a neutron product.